bls/globals.go

```diff
@@ -150,3 +150,15 @@ func EvaluatePolyInEvaluationForm(yFr *Fr, poly []Fr, x *Fr, rootsOfUnity []Fr,
 	// (x**WIDTH - 1) / WIDTH  *  sum_(i=0)^WIDTH  (f(DOMAIN[i]) * DOMAIN[i]) / (x - DOMAIN[i])
 	MulModFr(yFr, &y, &tmp)
 }
+
+func PolyLinComb(vectors [][]Fr, scalars []Fr) []Fr {
+	r := make([]Fr, len(vectors[0]))
```

Comment:
Should we have a `len(vectors) == 0` check here? The spec doesn't document what the result should be if the input is empty.

Reply:
good catch, yes we don't want a panic in case that happens.  Not sure it's worth adding an error return value in case that happens, so how about we just document it as returning []Fr{} in this case?

Reply:
sgtm

Reply:
Hmm, there's also the case where one of the other vectors has a different length, which would either result in panic or a result that doesn't make sense.  If we check for that case maybe we should just make it return error?

Reply:
Made it return error. ptal?